### DISCUSSION TITLE
On change anything on the page giving error. Data bind also not working. After changing value in the text area giving error Undefined offset: 1 app/Providers/AppServiceProvider.php:222

### DISCUSSION THREAD

**Discussion by DebasisBehera123** [livewire/livewire#4427] [1. Help] [answered]
HI
Can anyone say why this is happening?
Just a simple count component.
Even the data binding also throws the same error when changing the value in the text box.
Any help would be appreciated.

**Comment by joshhanley** [comment, marked as answer]
@DebasisBehera123 can you share a screenshot of the error message as well as your component class and view plus your route definition so we can have a look?

Thanks!

**Reply by DebasisBehera123** [reply]
Thank you [joshhanley](https://github.com/joshhanley) for your reply.
I fixed that bug.
Actually, the problem was in my AppServiceProvider.php not in livewire.
Thanks

**Reply by joshhanley** [reply]
No worries!
